tldw_Server_API/app/main.py

```
@@ -2519,6 +2519,16 @@ async def _run_heavy_initializations(*, deferred: bool) -> None:
         if deferred:
             logger.info("Deferred startup: completed non-critical initializations")
 
+    # Load persona archetypes and MCP server catalog (lightweight YAML reads).
+    try:
+        from tldw_Server_API.app.core.Persona.archetype_loader import load_archetypes_from_directory
+        from tldw_Server_API.app.core.MCP_unified.catalog_loader import load_mcp_catalog
+
+        load_archetypes_from_directory("tldw_Server_API/Config_Files/persona_archetypes")
+        load_mcp_catalog("tldw_Server_API/Config_Files/mcp_server_catalog.yaml")
+    except _STARTUP_GUARD_EXCEPTIONS as _archetype_err:
+        logger.debug(f"Archetype/catalog loading skipped: {_archetype_err}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)